chrome-extension/background.js

```diff
@@ -0,0 +1,12 @@
+chrome.runtime.onInstalled.addListener(function() {
+  chrome.declarativeContent.onPageChanged.removeRules(undefined, function() {
```

Comment:
nit: removeRules(undefined, callback) confused me initially, but now I know passing in `undefined` removes all rules. Up to you if you want to add a short comment or not :)